python-heasoft-osa/entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export HOME_OVERRRIDE=/tmp/jovyan; 
```

Comment:
this is different than what is set in the `Dockerfile`

Reply:
We need a writable directory, it differs. Anyway, I removed the during the build.
Maybe home will work too, we'll try.